greenmail-core/src/test/java/com/icegreen/greenmail/examples/ExampleReceiveNoRuleTest.java

```diff
@@ -1,22 +1,23 @@
 package com.icegreen.greenmail.examples;
 
+import static org.junit.Assert.assertEquals;
+
+import java.io.IOException;
+import javax.mail.MessagingException;
+import javax.mail.internet.MimeMessage;
+
 import com.icegreen.greenmail.user.GreenMailUser;
 import com.icegreen.greenmail.util.GreenMail;
 import com.icegreen.greenmail.util.GreenMailUtil;
 import com.icegreen.greenmail.util.ServerSetupTest;
 import org.junit.Test;
 
-import javax.mail.MessagingException;
```

Comment:
Sure, I will do this, can I ask you why?

Again, this was done automatically by my IDE, I never give much attention to the import-statement-block normally so I would be interested why this is important for you?